sql/core/src/test/resources/sql-tests/analyzer-results/identifier-clause.sql.out

```diff
@@ -850,10 +850,11 @@ VALUES(IDENTIFIER('a.b.c.d')())
 -- !query analysis
 org.apache.spark.sql.AnalysisException
 {
-  "errorClass" : "IDENTIFIER_TOO_MANY_NAME_PARTS",
-  "sqlState" : "42601",
+  "errorClass" : "REQUIRES_SINGLE_PART_NAMESPACE",
```

Comment:
The reason for modifying these two golden files is that:
when using V2SessionCatalog, `resolveV1Function` was used previously, which would throw the exception `IDENTIFIER_TOO_MANY_NAME_PARTS`. 
After modification, it follows the V2 logic and throws the V2 exception `REQUIRES_SINGLE_PART_NAMESPACE`.
 
The latter behavior is more reasonable, as exemplified by lines 757-767.

```
-- !query
CREATE TABLE IDENTIFIER('a.b.c')(c1 INT) USING csv
-- !query analysis
org.apache.spark.sql.AnalysisException
{
  "errorClass" : "REQUIRES_SINGLE_PART_NAMESPACE",
  "sqlState" : "42K05",
  "messageParameters" : {
    "namespace" : "`a`.`b`",
    "sessionCatalog" : "spark_catalog"
  }
}
```